src/express-middleware.js

```diff
@@ -57,11 +57,16 @@ class ExpressMiddleware {
                 route = route ? route + req.route.path : req.route.path;
             }
 
+            let originalUrl = req.originalUrl;
```

Comment:
I believe that there is another case that isn't covered even with your fix. when the url contains multiple consecutive `/`, i.e: `GET /v1//test`

what do you think about this kind of an handling
```js
            if (typeof route !== 'string') {
                route = req.originalUrl.split('?')[0];
            } else {
                const splittedRoute = route.split('/').filter(seg => !!seg);
                const splittedUrl = req.originalUrl.split('?')[0].split('/').filter(seg => !!seg);
                const routeIndex = splittedUrl.length - splittedRoute.length;

                route = '/' + splittedUrl.slice(0, routeIndex).concat(splittedRoute).join('/');
            }
```